samples/openvx/openvx_interoperability.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+#include <VX/vx_lib_debug.h>
+#include <VX/vx_helper.h>
+#include <VX/vx_lib_extras.h>
+
+//for vx-Access/Commit-ImagePatch
+#include <VX/vx_compatibility.h>
```

Comment:
Why are are using Access/Commit instead of new Map/Unmap/Copy methods? The Access/Commit are deprecated in OpenVX 1.1.